x-pack/plugins/security_solution/cypress/e2e/detection_alerts/investigate_in_timeline.cy.ts

```diff
@@ -64,17 +65,24 @@ describe('Investigate in timeline', { testIsolation: false }, () => {
 
     it('should open a new timeline from a prevalence field', () => {
       cy.get(SUMMARY_VIEW_PREVALENCE_CELL)
-        .first()
+        .last()
         .invoke('text')
         .then((alertCount) => {
-          // Click on the first button that lets us investigate in timeline
-          cy.get(ALERT_FLYOUT).find(SUMMARY_VIEW_INVESTIGATE_IN_TIMELINE_BUTTON).first().click();
+          // Click on the last button that lets us investigate in timeline.
+          // We expect this to be the `process.args` row.
+          cy.get(ALERT_FLYOUT).find(SUMMARY_VIEW_INVESTIGATE_IN_TIMELINE_BUTTON).last().click();
```

Comment:
When possible, it is better to have this type of methods wrapped into a task method.